logging/interceptor.go

```diff
@@ -109,12 +135,42 @@ func UnaryServerInterceptor(logger *logrus.Logger, opts ...Option) grpc.UnarySer
 		levelLogf(
 			ctxlogrus.Extract(newCtx).WithFields(fields),
 			options.codeToLevel(code),
-			"finished unary call with code "+code.String())
+			fmt.Sprintf("finished unary call with code %s", code.String()))
```

Comment:
Per definition of levelLogf ```func levelLogf(entry *logrus.Entry, level logrus.Level, format string, args ...interface{}) {``` we can add arguments directly into this function instead of using fmt.Sprintf